app/views/login/login.html.haml

```diff
@@ -1,3 +1,9 @@
+- flash.each do |key, value|
+  .alert{ :class => "alert-#{key}" }
```

Comment:
It looks like you're referencing some CSS classes that we don't have in our app currently. We have some existing CSS that you might be able to hook onto in `app/gulp/sass/_error.scss`, or you could add your own.